src/components/file-upload/index.md

```diff
@@ -112,46 +114,57 @@ Say ‘The selected file must use the template’.
 
-Although, we recommend service teams start using the improved component to improve the experience for users, it is a substantial visual change. To allow teams to migrate at their own pace, the improvements are behind a ‘feature flag’.
+This example shows you how to enable the improved File upload component:
 
 {{ example({ group: "components", item: "file-upload", example: "enhanced", html: true, nunjucks: true, open: false, size: "m" }) }}
 
 ### Changes in the improved component
 
-To make it easier for users to ‘drag and drop’ files, we’ve made the drop zone:
+To make it easier for users to drag and drop files, we’ve made the drop zone:
 
 - bigger
 - visible at all times
 - more visually responsive to user interactions
 
-We’ve also changed the ‘Choose file’ button to be more consistent with the secondary button in the [Button component](/components/button/).
+We’ve also changed the ‘Choose file’ button to be more consistent with the secondary button in the [Button component](/components/button).
 
-You can change the text on the button and the ‘no file chosen’ message.
+Service teams can change the text on the button and in the ‘no file chosen’ message. We decided to make this text changeable for translation purposes and to let teams be specific about the file to upload. However, teams should aim to keep the text as short as possible for accessibility purposes. For example, screen reader users might find it difficult to use the component if the text is too long.
 
-All the text in the component can now be translated to match the language the page is in when JavaScript is running.
+All the text in the component can now be translated to match the language of the page content when JavaScript is running.
 
 #### Improvements for assistive technology users
 
-Users of Dragon Naturally Speaking, a speech recognition tool, will now be able to activate the component by saying commands for interacting with [web page controls](https://www.nuance.com/products/help/dragon/dragon-for-pc/enx/professionalgroup/main/Content/Web/working_with_chrome.htm?Highlight=click%20button). This feature currently does not work with native file input from the browser.
+Users of Dragon, a speech recognition tool, [cannot activate their browser’s native file inputs](https://github.com/alphagov/govuk-frontend/issues/3686) by using commands for interacting with [web page controls](https://www.nuance.com/products/help/dragon/dragon-for-pc/enx/professionalgroup/main/Content/Web/working_with_chrome.htm?Highlight=click%20button). They have to rely on [mouse commands](https://www.nuance.com/products/help/dragon/dragon-for-pc/enx/professionalgroup/main/Content/CommandandControl/using_your_mouse.htm) or [keyboard commands](https://www.nuance.com/products/help/dragon/dragon-for-pc/enx/professionalgroup/main/Content/CommandandControl/using_your_keyboard.htm), which take multiple steps to activate the component.
```

Comment:
@selfthinker Can you confirm this is the right issue, please? The [doc we used to review the guidance (internal)](https://docs.google.com/document/d/1Ogww7PYCWGKzV09-f6LlzjltQZbMUpmtHR9EHOPhKG8/edit?tab=t.0#heading=h.c3aafm6j8d1l) had this link visible, but the link led to alphagov/govuk-frontend#2425 (probably due to the resolution of comments on the doc).

Reply:
The link in the PR is the right one after @seaemsi double checked the comments, so I'll mark as resolved 😊